Classes/BEMAnalogClockView.m

```diff
@@ -260,7 +261,12 @@ - (void)updateTimeAnimated:(BOOL)animated {
 - (void)setClockToCurrentTimeAnimated:(BOOL)animated {
     NSDate *currentTime = [NSDate date];
     NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
-    [dateFormatter setDateFormat:@"hh"];
+    if (self.militaryTime == NO) {
+        [dateFormatter setDateFormat:@"hh"];
+    }
+    else {
```

Comment:
Nit: the rest of the project uses `} else {`. For consistency, please conform to this style.
There are multiple occurrences of this, but I'm not commenting here once.